config/initializers/custom_mime_types.rb

```diff
@@ -0,0 +1 @@
+Mime::Type.register 'application/rdf+xml', :rdf
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.